rest/src/test/java/com/carhouse/rest/controllerIT/CarControllerTestIT.java

```diff
@@ -102,13 +105,14 @@ void updateCarWithWrongReference() {
     @Test
     void updateNotExistCar() {
         int carId = 1;
+        int carSaleId = 120;
         Car car = restTemplate.getForObject(HOST + CAR_GET_URL + carId, Car.class);
-        car.setCarId(120);
+        car.setCarId(carSaleId);
         HttpEntity<Car> request = new HttpEntity<>(car);
         HttpClientErrorException exception = assertThrows(HttpClientErrorException.class,
                 () -> restTemplate.exchange(HOST + CAR_UPDATE_URL, HttpMethod.PUT, request, String.class));
         assertEquals(HttpStatus.NOT_FOUND, exception.getStatusCode());
-        assertTrue(exception.getResponseBodyAsString().contains("there is not car with id = " + 120));
+        assertTrue(exception.getResponseBodyAsString().contains("there is not car with id = " + carSaleId));
```

Comment:
Looks like here a lot of validations. Anywhere checked existence of 4 fields in the response.

Reply:
> Looks like here a lot of validations
What does it mean? What should I do?
I checked what the response has 4 fields using regular expression is it right?